cli/src/torii/routing.rs

```diff
@@ -426,7 +425,7 @@ async fn handle_status(sumeragi: Arc<Sumeragi>, _network: Addr<IrohaNetwork>) ->
     if let Err(error) = sumeragi.update_metrics() {
         iroha_logger::error!(%error, "Error while calling `sumeragi::update_metrics`.");
     }
-    let status = Status::from(&sumeragi.wsv_mutex_access().metrics);
+    let status = Status::from(&sumeragi.metrics_mutex_access());
```

Comment:
Please consider using atomics.

Reply:
I quite strongly disagree. You can't lock atomics. They can be changed from elsewhere while you are reading them.

Reply:
You don't want another query to partially update the metrics while you are reading from them.